recipes/libcurl/all/conanfile.py

```diff
@@ -104,14 +111,19 @@ def system_requirements(self):
         #   would be much better than installed them (sudo required).
         pass
 
+    def build_requirements(self):
+        if tools.os_info.is_windows:
+            self.build_requires("ninja/[>=1.9.0]")
```

Comment:
```suggestion
            self.build_requires("ninja/1.9.0")
```

Do not use ranges in CCI.